### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rombur**
The only changes in the `#### Scan-path-based deposition` section is the removal of extra `\`. The text itself is unchanged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +55,19 @@ using
 ```bash
 docker pull rombur/adamantine:latest
 ```
+The 1.0 release version is available using
+```bash
+docker pull rombur/adamantine:1.0
+```
+To start an interactive container use
+```
+docker run --rm -it rombur/adamantine:latest bash
```
Comment (stvdwtt) [inline README.md:64]: Should this be?

```suggestion
docker run --rm -it rombur/adamantine:1.0 bash
```

File: README.md [hunk 2]
```diff
@@ -123,17 +147,17 @@ The following options are available:
 * materials (required):
   * n\_materials: number of materials (required)
   * property\_format: format of the material property: table or polynomial (required)
-  * initial\_temperature: initial temperature of all the materials (default value: 300)
-  * new\_material\_temperature: temperature of all the material that is being added during the process (default value: 300)
+  * initial\_temperature: initial temperature of all the materials in kelvins (default value: 300)
```
Comment (stvdwtt) [inline README.md:150]:
```suggestion
  * initial\_temperature: initial temperature of all the materials in Kelvin (default value: 300)
```

File: README.md [hunk 3]
```diff
@@ -123,17 +147,17 @@ The following options are available:
...
+  * new\_material\_temperature: temperature of all the material that is being added during the process in kelvins (default value: 300)
```
Comment (stvdwtt) [inline README.md:151]:
```suggestion
  * new\_material\_temperature: temperature of all the material that is being added during the process in Kelvin (default value: 300)
```

File: README.md [hunk 4]
```diff
@@ -123,17 +147,17 @@ The following options are available:
...
   (one is required)
   * material\_X.Y.Z: Z is either density in kg/m^3, specific\_heat in J/(K\*kg),
   thermal\_conductivity\_x, resp. y or z, in the direction x, resp. y or z (in 2D only x and z are used), in W/(m\*K), emissivity,
   or convection\_heat\_transfer\_coef in W/(m^2\*K) (optional)
-  * material\_X.A: A is either solidus in kelvin, liquidus in kelvin, latent\_heat
-  in J/kg, radiation\_temperature\_infty in kelvin, or convection\_temperature\_infty
-  in kelvin (optional)
+  * material\_X.A: A is either solidus in kelvins, liquidus in kelvins, latent\_heat
+  in J/kg, radiation\_temperature\_infty in kelvins, or convection\_temperature\_infty
+  in kelvins (optional)
```
Comment (stvdwtt) [inline README.md:160]:
```suggestion
  * material\_X.A: A is either solidus in Kelvin, liquidus in Kelvin, latent\_heat
  in J/kg, radiation\_temperature\_infty in Kelvin, or convection\_temperature\_infty
  in Kelvin (optional)
```

File: README.md (hunk 2 above)
Comment (Rombur) [reply README.md:150]: I wondered about this. My understanding is that `kelvin` is a unit and so it should be treated the same way as meters. There is this rule on wikipedia https://en.wikipedia.org/wiki/Kelvin#Orthography but if you think it should be `Kelvin`, I'll change it.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,16 @@
-![image](doc/logo/adamantine_wordmark.png)
+![image](doc/logo/adamantine_wordmark_with_bg.png)
 
 `adamantine` stands for *AD*ditive (*A*) *MAN*unifac*T*uring s*I*mulator (*NE*).
-It is an open-source sofware to simulate heat transfer for additive manufacturing.
+`adamantine` is a thermomechanical code for additive manufacturing. It is based
```
[the remark this answers is not in the document]
Comment (Rombur) [reply README.md:4]: Sure I'll change it

File: README.md (hunk 2 above)
Comment (stvdwtt) [reply README.md:149]: Oh, it looks like I've been using it incorrectly for years. You were right and it should be `kelvin` and `kelvins` like you had it.

File: README.md (hunk 3 above)
Comment (stvdwtt) [reply README.md:150]: Correct as-is (see above).

File: README.md (hunk 4 above)
Comment (stvdwtt) [reply README.md:159]: Correct as-is, see above.

**Review by stvdwtt — APPROVED** [review]

**Comment by Rombur** [comment]
Since this PR only touches the README, we don't need to wait for the CI. I am merging it.

**Pull request merged**